participation/membership-tiers.md

```diff
@@ -1,2 +1,9 @@
 |Corda Network Foundation|[Document history]({{ site.github.repository_url }}/blame/master/{{page.path}})|
```

Comment:
can we turn this into a comparative table, as you did with the powerpoint? and add the same level of detail?

Reply:
yes. done. I have added in the pricing - good idea or not? I think it's okay.